### PR TITLE
Draw circle different

### PR DESCRIPTION
**Pull request by Buskcoin**
This is similar to my last pr but with a complete circle. The advantage of this is that the circle will react to corners and in the future we can add a wiggle function that will modify all the points. The amount of points drawn through is directly related to mass. I still haven't applied this to the enemy player but I'll do that next if this is merged.

### CODE REVIEW

**Comment by abalabahaha** [comment]
From my testing, adding a new point every time the player feasts kinda looks bad...

**Comment by Buskcoin** [comment]
Is that better?

File: client/js/app.js [hunk 1]
```diff
@@ -561,6 +554,11 @@
         }
     }
 
+    function contain(min, max, value) {
+        return Math.min(max, Math.max(min, value));
```
Comment (bigfoot90) [inline client/js/app.js]: Is this function returning ```True``` if ```value``` is between ```min``` and ```max```?
If this is wanted you should rename the function.

Comment (abalabahaha) [inline client/js/app.js]: @bigfoot90  this basically "contains" the value inside the min-max range. If it's smaller than min, it returns min. If it's bigger than max, it returns max. This is to ensure "value" doesn't go out of the bounds specified.

Comment (bigfoot90) [inline client/js/app.js]: I suggest ```ensureValueInRange``` as more descriptive.
*Better code = Less errors = Faster development*

**Comment by Buskcoin** [comment]
Ok, This is the function that makes the blob change shape around the border. How about 'outOfBounds'.

**Comment by bigfoot90** [comment]
@Buskcoin ```outOfBounds``` indicate something did *out*. Still prefer ```ensureValueInRange```.
Semantics improve code readbility.

**Pull request merged**